README.md

```diff
@@ -37,7 +37,7 @@ If you choose this option, you can skip next the step which generates Godot head
 - To generate Godot headers, you need to run the compilation command inside the `godot` submodule directory:
 
 ```bash
-scons platform=iphone target=debug
+scons platform=<iphone|ios> target=debug

```

Comment:
I'd just use `ios` only.
Not all users are familiar with this kind of formatting to show possible arguments and might just copy paste as is and get an error.

Reply:
Ah I guess it's different for `3.x` and 4.0.
Maybe:
```suggestion
# For Godot 3.x:
scons platform=iphone target=debug
# For Godot 4.x:
scons platform=ios target=debug
```

Reply:
Updated:
<img width="864" alt="Screenshot 2022-07-21 at 11 34 28" src="https://user-images.githubusercontent.com/7645683/180169046-d1c31db8-eba3-4c58-b7d5-fac43ca18398.png">